src/components/RootNavigator/SwapNavigator.js

```diff
@@ -42,56 +42,57 @@ export default function SwapNavigator() {
         component={SwapFormSelectCrypto}
         options={{
           headerTitle: () => <StepHeader title={t("transfer.swap.title")} />,
-          headerRight: null,
+          headerRight: undefined,
```

Comment:
Can you remove these lines? It should work without it if you set default screen options with Stack.Navigator comp.